packages/@aws-cdk/aws-eks/README.md

```diff
@@ -176,6 +176,32 @@ cluster.addNodegroup('nodegroup', {
 });
 ```
 
+### ARM64 Support
+
+Instance types with `ARM64` architecture are supported in both managed nodegroup and self-managed nodegroup. Simply specify the `instanceType` and the latest Amazon Linux 2 AMI for ARM64 will be selected.
+
+```ts
+// create a cluster with a default managed nodegroup 
+cluster = new eks.Cluster(this, 'Cluster', {
+  vpc,
+  mastersRole,
+  defaultCapacity: 2,
+  defaultCapacityInstance: new ec2.InstanceType('m6g.medium'),
```

Comment:
Kind of redundant since you have two examples below already for node groups and capacity:

```suggestion
```